forge-gui/res/cardsfolder/upcoming/oblivious_bookworm.txt

```diff
@@ -0,0 +1,14 @@
+Name:Oblivious Bookworm
+ManaCost:G U
+Types:Creature Human Wizard
+PT:2/3
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | OptionalDecider$ You | TriggerZones$ Battlefield | Execute$ TrigDraw | TriggerDescription$ At the beginning of your end step, you may draw a card. If you do, discard a card unless a permanent entered the battlefield face down under your control this turn or you turned a permanent face up this turn.
+SVar:TrigDraw:DB$ Draw | SubAbility$ DBDiscard
+SVar:DBDiscard:DB$ Discard | Defined$ You | NumCards$ 1 | Mode$ TgtChoose | ConditionCheckSVar$ Condition | ConditionSVarCompare$ EQ0
+T:Mode$ TurnFaceUp | ValidCard$ Permanent.YouCtrl | Execute$ TrigRemember | TriggerZones$ Battlefield | Static$ True
```

Comment:
check for ValidCause instead

Reply:
does it ignore stuff like via _Clone Shell_ yet?

Reply:
Should it ignore it ? It did imprint it  
![image](https://github.com/user-attachments/assets/f763035c-6826-4fe9-b97d-8b2add8009a1)

Reply:
yea
add `+inZoneBattlefield`